src/libraries/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/Marshal/AddRefTests.cs

```diff
@@ -1,17 +1,18 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+using System.Runtime.InteropServices.Tests.Common;
 using Xunit;
 
 namespace System.Runtime.InteropServices.Tests
 {
     public class AddRefTests
     {
         [Fact]
-        [PlatformSpecific(TestPlatforms.Windows)]
```

Comment:
We still need to condition running to only with `ComWrappers` support / on coreclr, right?

Reply:
Good point. That would explain the failures. I'd not started investigating until I hear back about the OS support question.